MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include delta_shooter/ *.png
```

Comment:
What does this do? Google tells me it adds these png files to the package when pip installed - is that right?

Reply:
yep, you need it because the game is dependent on the assets folder for the sprites. Normally a pip install wouldn't include sub-directories full of pngs :D